engine/evd.mli

```diff
@@ -31,7 +31,7 @@ open Environ
 (** {6 Evars} *)
 
 type evar = existential_key
-(** Existential variables. TODO: Should be made opaque one day. *)
+(** Existential variables. *)
```

Comment:
Why not to make it opaque now?

Reply:
It is opaque, this comment is from before 33eea163c72c70eaa3bf76506c1d07a8cde911fd (make Evar.t opaque). At least I assume that's what @ppedrot meant when he wrote it in 0737d090ed32c0857757b76dc94bb5aaa1a096ef.

Reply:
Ah, right, I missed it.